vtkLookingGlassRenderWindowImpl.h

```diff
@@ -88,3 +88,33 @@ void className::SetLGDeviceIndex(int index)
 {
   this->Interface->SetDeviceIndex(index);
 }
+
+//------------------------------------------------------------------------------
+void className::SaveQuilt(const char* fileName)
+{
+  this->Interface->SaveQuilt(fileName);
+}
+
+//------------------------------------------------------------------------------
+void className::StartRecordingQuilt(const char* fileName)
+{
+  this->Interface->StartRecordingQuilt(fileName);
+}
+
+//------------------------------------------------------------------------------
+void className::StopRecordingQuilt()
+{
+  this->Interface->StopRecordingQuilt();
+}
+
+//------------------------------------------------------------------------------
+const char* className::MovieFileExtension()
+{
+  return vtkLookingGlassInterface::MovieFileExtension();
+}
+
+//------------------------------------------------------------------------------
+std::string className::QuiltFileSuffix() const
```

Comment:
I seems that we have some methods that return `std::string` and some that return `const char*`. Should we be consistent and just use `std::string`?